modules/mauth-signer-akka-http/src/main/scala/com/mdsol/mauth/http/Implicits.scala

```diff
@@ -32,7 +33,10 @@ object Implicits {
       * Create an akka-http request from a [[com.mdsol.mauth.models.SignedRequest]]
       */
     def toAkkaHttpRequest: HttpRequest = {
-      val allHeaders = (signedRequest.req.headers ++ signedRequest.mauthHeaders).toList
+      val contentType: Option[String] = extractContentTypeFromHeaders(signedRequest.req.headers)
+      val headersWithoutContentType: Map[String, String] = removeContentTypeFromHeaders(signedRequest.req.headers)
```

Comment:
Don't need this as akka-http will remove it anyway. (Will be good to add a comment though)

Reply:
Nevermind, probably less confusing to remove it outright